mirage/scenarios/default.js

```diff
@@ -27,7 +27,7 @@ export default function(server) {
   })
 
   server.createList('activity', 3, { userId: user.id })
-  server.create('activity', 'active', { userId: user.id })
+  // server.create('activity', 'active', { userId: user.id })
```

Comment:
Why disable this?

Reply:
I'll probably better remove this line altogether.

The active activity is created inside the test(s). Having one in the scenario breaks the tests. No idea why this worked before / what changed.

Reply:
Mirage doesn't magically load scenarios in the tests.. This scenario will only be loaded when you start the server without proxy. This should not affect the tests - if it does, something else is wrong..

Reply:
You're right! The test still pass.

Looks like I made more than one change at once when I was debugging the tests.

Thank you!